### PR TITLE
Update: ListPicker: allow linkButtons to accept nodes

### PR DESCRIPTION
**Pull request by lightbringer1991**

### CODE REVIEW

**Comment by lightbringer1991** [comment]
[(3) Create product: tabs & paging](https://trello.com/c/LPb3SmDZ/6526-3-create-product-tabs-paging)

File: src/components/adslotUi/ListPickerComponent.jsx [hunk 1]
```diff
@@ -135,11 +135,11 @@ class ListPickerComponent extends React.Component {
         <Modal.Footer>
           {_.isEmpty(props.linkButtons) ? null :
             <div className="pull-left">
-              {_.map(props.linkButtons, (linkButton) =>
+              {_.map(props.linkButtons, (linkButton) => (React.isValidElement(linkButton) ? linkButton : (
```
Comment (omgaz) [inline src/components/adslotUi/ListPickerComponent.jsx]: You don't need to do isValidElement if you use the propType `PropTypes.element`

File: src/components/adslotUi/ListPickerComponent.jsx [hunk 2]
```diff
@@ -163,6 +163,14 @@ const itemProps = PropTypes.shape({
   id: PropTypes.number.isRequired,
 });
 
+const linkButtonsProps = PropTypes.arrayOf(PropTypes.oneOfType([
+  PropTypes.shape({
+    label: PropTypes.string.isRequired,
+    href: PropTypes.string.isRequired,
+  }),
+  PropTypes.node,
```
Comment (omgaz) [inline src/components/adslotUi/ListPickerComponent.jsx]: see above

File: src/components/adslotUi/ListPickerComponent.jsx (hunk 1 above)
Comment (lightbringer1991) [inline src/components/adslotUi/ListPickerComponent.jsx]: hmm I actually want it to be either a node, or an object with predefined attributes. I'll change the code to reflect that

File: src/components/adslotUi/ListPickerComponent.jsx [hunk 3]
```diff
@@ -12,6 +12,11 @@ import SvgSymbol from 'components/alexandria/SvgSymbol';
 
 require('styles/adslotUi/ListPicker.scss');
 
+const isSubset = (array, subArray) =>
```
Comment (lightbringer1991) [inline src/components/adslotUi/ListPickerComponent.jsx]: I'm not sure if lodash already have a function to check if an array is a subset of another array. I'm happy to switch if it does

tried `_.includes` but it seems to only check for 1 element

**Comment by omgaz** [comment]
LGTM

File: src/components/adslotUi/ListPickerComponent.jsx (hunk 3 above)
Comment (omgaz) [inline src/components/adslotUi/ListPickerComponent.jsx]: Maybe _(subarray).without(array).isEmpty()?

**Comment by lteacher** [comment]
merging

**Pull request merged**